script.js

```diff
@@ -0,0 +1,17 @@
+const burger = document.getElementById('burger');
+const menuItems = document.getElementById('menu-block');
+const close = document.getElementById('close');
+const item1 = document.getElementById('menuitem1');
+const item2 = document.getElementById('menuitem2');
+const item3 = document.getElementById('menuitem3');
```

Comment:
- [ ] [OPTIONAL] To avoid repetition in your coded and use best practices it's better to use ```querySelectorAll()``` instead to select all the menu links at ones